src/conduit/ConduitClient.php

```diff
@@ -163,6 +184,55 @@ public function callMethod($method, array $params) {
     return $conduit_future;
   }
 
+  // UBER CODE BEGIN
+  private function addWonkaClaimToCookies($cookies) {
+    $claim = $this->getWonkaClaim();
+    $new_cookies = '';
+    if (empty($claim)) {
+      return $cookies;
+    }
+    $usso_cookie = "usso={$claim}";
+    if (empty($cookies)) {
+      $new_cookies = $usso_cookie;
+    } else {
+      $new_cookies = $cookies."; {$usso_cookie}";
+    }
+    return $new_cookies;
+  }
+
+  private function getWonkaClaim() {
+    $claim = '';
+    // if the file exists and is current, use it.
+    $claim_file = getenv('ARC_USSO_COOKIE');
+    if (file_exists($claim_file)) {
+      $stale = time() - filemtime($claim_file);
+      if ($stale < 60) {
+        $claim = $this->readClaimFile($claim_file);
+      }
+    }
+    if (empty($claim)) {
+      $wonka_cli = getenv('ARC_WONKA_CLI');
+      system("{$wonka_cli} {$claim_file}");
```

Comment:
This is security flaw. I can provide anything via `ARC_USSO_COOKIE` and `ARC_WONKA_CLI`.